.github/workflows/if-nodejs-release.yml

```
@@ -47,7 +47,7 @@ jobs:
 
   release:
     needs: test
-    name: Publish to NPM and GitHub
+    name: Publish to NPM/DockerHub and GitHub
```

Just small suggestion that makes sense to me (but can be obviously discarded)
```suggestion
    name: Publish to any of NPM, Github, and Docker Hub
```